app.js

```diff
@@ -12,47 +12,27 @@ $('a.designSprintFullStudyReadLink').on('click', function() {
 	$( 'div.designSprintFullStudy' ).toggleClass( 'show' );
```

Comment:
@talitore 
Hi! So I'm having some problems over here and I'm not sure of the best way to approach this. What I'd like to happen is when I click on the "Read" link, the hidden content which is all of the project info expands on the page and the "Read" link then says "Close". Right now all of the links change to say "Close" and never change back to "Read". 
Basically I want it to act as an accordion, but I can't seem to make it work. What do you suggest?